esp/src/src-react/components/Common.tsx

```diff
@@ -1,4 +1,24 @@
 import * as React from "react";
 import { VerticalDivider } from "@fluentui/react";
+import * as Utility from "src/Utility";
+import nlsHPCC from "src/nlsHPCC";
 
 export const ShortVerticalDivider = () => <VerticalDivider styles={{ divider: { paddingTop: "20%", height: "60%" } }} />;
+
+export function createCopyDownloadSelection(grid, selection: any, filename: string) {
```

Comment:
What is `grid`? It should probably be typed.

Reply:
grid is a DGrid and doesn't have a type.